src/sentry/models/projectownership.py

```diff
@@ -251,6 +251,11 @@ def handle_auto_assignment(cls, project_id, event):
             User,
         )
 
+        force_autoassign = True
```

Comment:
Could you please add a comment to this block? It's hard to follow as to why an event would cause auto assignment or not.